### PR TITLE
Implemented NMS.FT.132, NMS.FT.133, NMS.FT.134 basically these test c…

### PR DESCRIPTION
**Pull request by PriyaJain5**
…ase checks

NMS is able to create new subscription for subscriber whose old subscription
is completed/deactivated
.

### CODE REVIEW

File: testing/src/test/java/org/motechproject/nms/testing/it/kilkari/SubscriptionServiceBundleIT.java [hunk 1]
```diff
@@ -793,6 +794,7 @@ public void verifyFT180() {
     /*
      * To verify LMP is changed successfully and new subscription created
      * when subscription already exist for 72Weeks Pack having status as "Completed".
+     * Now Added one more assert for updated LMP to cover NMS_FT_134.
```
Comment (frankhuster) [inline testing/src/test/java/org/motechproject/nms/testing/it/kilkari/SubscriptionServiceBundleIT.java]: Please add text description of NMS_FT_134

File: testing/src/test/java/org/motechproject/nms/testing/it/kilkari/SubscriptionServiceBundleIT.java [hunk 2]
```diff
@@ -793,6 +794,7 @@ public void verifyFT180() {
     /*
      * To verify LMP is changed successfully and new subscription created
      * when subscription already exist for 72Weeks Pack having status as "Completed".
```
Comment (frankhuster) [inline testing/src/test/java/org/motechproject/nms/testing/it/kilkari/SubscriptionServiceBundleIT.java]: please use childPack or pregnancyPack

File: testing/src/test/java/org/motechproject/nms/testing/it/kilkari/SubscriptionServiceBundleIT.java [hunk 3]
```diff
@@ -828,6 +834,7 @@ public void verifyFT156() {
     /*
      * To verify DOB is changed successfully and new subscription created
      * when subscription already exist for 48Weeks Pack having status as "Deactivated".
```
Comment (frankhuster) [inline testing/src/test/java/org/motechproject/nms/testing/it/kilkari/SubscriptionServiceBundleIT.java]: use childPack or pregnancyPack

File: testing/src/test/java/org/motechproject/nms/testing/it/kilkari/SubscriptionServiceBundleIT.java [hunk 4]
```diff
@@ -828,6 +834,7 @@ public void verifyFT156() {
...
+     * Now Added one more assert for updated DOB to cover NMS_FT_132
```
Comment (frankhuster) [inline testing/src/test/java/org/motechproject/nms/testing/it/kilkari/SubscriptionServiceBundleIT.java]: Include text description of NMS_FT_132

File: testing/src/test/java/org/motechproject/nms/testing/it/kilkari/SubscriptionServiceBundleIT.java [hunk 5]
```diff
@@ -855,6 +870,7 @@ public void verifyFT159() {
     /*
      * To verify DOB is changed successfully and new subscription created
      * when subscription already exist for 48Weeks Pack having status as "Completed".
```
Comment (frankhuster) [inline testing/src/test/java/org/motechproject/nms/testing/it/kilkari/SubscriptionServiceBundleIT.java]: use childPack

File: testing/src/test/java/org/motechproject/nms/testing/it/kilkari/SubscriptionServiceBundleIT.java [hunk 6]
```diff
@@ -855,6 +870,7 @@ public void verifyFT159() {
...
+     * Now Added one more assert for updated DOB to cover NMS_FT_133
```
Comment (frankhuster) [inline testing/src/test/java/org/motechproject/nms/testing/it/kilkari/SubscriptionServiceBundleIT.java]: text description for NMS_FT_133

**Comment by frankhuster** [comment]
a few small fixes

**Comment by PriyaJain5** [comment]
Description has been added in test-case. Below is summary for test description: 
Earlier we were checking whether subscription is created or not after deactivation/completion. But we need some more assert to check LMP/DOB is updating accordingly or not.

**Comment by PriyaJain5** [comment]
As this branch has been merged in SubscriptionServiceBundleIT branch

**Pull request closed without merging**